.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: "3.1"
+          ruby-version: "3.4"
```

Comment:
We should keep CI on Ruby 3.1 for now, since the actual bazel job still runs Ruby 3.1. (We do need to update the Bazel configs to the Ruby 3.4 runtime as well, but that also might require us to update actionpack, for which I have a WIP branch.

Reply:
ack